sass/DynamicHierarchicalFacet/_DynamicHierarchicalFacetValues.scss

```diff
@@ -88,21 +88,33 @@ button.coveo-dynamic-hierarchical-facet-all {
   color: $color-blueish-gray;
 }
 
-.coveo-dynamic-hierarchical-facet-value-arrow {
-  position: absolute;
-  top: -1px;
-  left: 0;
+$arrow-svg-dimensions: 11px;
+.coveo-dynamic-hierarchical-facet-value-arrow-right,
+.coveo-dynamic-hierarchical-facet-value-arrow-left {
+  svg {
+    width: $arrow-svg-dimensions;
+    height: $arrow-svg-dimensions;
+    color: $coveo-blue;
+
+    display: inline-block;
+    vertical-align: middle;
+  }
 }
 
-$arrow-svg-dimensions: 14px;
-.coveo-dynamic-hierarchical-facet-value-arrow-svg {
-  transform: rotate(90deg);
-  width: $arrow-svg-dimensions;
-  height: $arrow-svg-dimensions;
-  color: $coveo-blue;
+.coveo-dynamic-hierarchical-facet-value-arrow-right {
+  margin-left: 8px;
+  svg {
+    transform: rotate(270deg);
+  }
+}
 
-  display: inline-block;
-  vertical-align: middle;
+.coveo-dynamic-hierarchical-facet-value-arrow-left {
+  position: absolute;
+  top: -2px;
```

Comment:
I'm guessing this `-2px` is for vertical alignment ? 

Could this have been achieved instead with a flex container with some `vertical-align: middle` or, if the container is flex, with `align-items: center` ?